src/components/webgl/WebglBox.tsx

```diff
@@ -17,7 +17,7 @@ function WebglBox({ userId, experienceData }): JSX.Element {
       // Having a timeout bypasses this
       setTimeout(() => {
         load(experienceData.apparatusId);
-      }, 100);
+      }, 5);
```

Comment:
is there a reason why timeout was changed here?

Reply:
I dont think so, changing back